apps/nextjs-website/react-components/components/HowTo/HowTo.tsx

```diff
@@ -86,26 +90,34 @@ const HowTo = (props: HowToProps) => {
 
         {/** Link */}
         {link && (
-          <Link
-            sx={{
-              display: 'flex',
-              alignItems: 'center',
-              mt: 6,
-              alignSelf: 'flex-start',
-            }}
-            href={link.href}
-            target={link.target}
-            color={textColor}
-            underline='none'
-            fontWeight={600}
+          <Typography
+            component='span'
+            variant='body1'
+            color={theme === 'light' ? 'text.primary' : 'background.paper'}
+            fontSize={'18px'}
```

Comment:
```suggestion
```
I believe the `body1` variant already has a fontSize of 18 so no need to set it again.